README.md

```diff
@@ -1,24 +1,14 @@
-# README
+# Memebers Only Application
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> Application to build an exclusive clubhouse where your members can write embarrassing posts about non-members. Inside the clubhouse, members can see who the author of a post is but, outside, they can only see the story and wonder who wrote it.
 
```

Comment:
For future projects it would also be nice to have sections here that explain how others can install the project. 

Another is a link to a Live Demo.

Reply:
Ok you are right, I didn't put the installation section because I don't how deep I need to go with the steps but I'm going to put it. About the Live Demo, I didn't put it because I'm almost full with the links on Heroku so I want to save some of them for other projects.